README.md

```diff
@@ -11,14 +11,17 @@ Before using this plugin, you must ensure that `gjslint` is installed on your sy
 Run `amp install linter-gjslint` or search for `linter-gjslint` in Atom package manager.
 
 ## Settings
-You can configure linter-gjslint by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
+You can configure linter-gjslint by accessing the options in File > Settings > Packages.<br>
+Or by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
 ```
```

Comment:
Specify `CoffeeScript` (or one of the aliases [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L569)) to enable syntax highlighting.